components/src/preact/mutationsOverTime/mutations-over-time-grid.tsx

```diff
@@ -141,9 +141,13 @@ const ProportionCell: FunctionComponent<{
                 <>
                     <p>Proportion: {formatProportion(value.proportion)}</p>
                     {value.count !== null && value.totalCount !== null && (
-                        <p>
-                            Count: {value.count} / {value.totalCount} total
-                        </p>
+                        <>
+                            <p>
+                                Count: {value.count} / {totalCountWithCoverage(value.count, value.proportion)} with
+                                coverage
+                            </p>
+                            <p>Total: {value.totalCount}</p>
```

Comment:
```suggestion
                            <p>{value.totalCount} in timeframe</p>
```
I like your suggestion in the info box so much that I think it would be nice here